includes/class-sensei-course.php

```diff
@@ -668,6 +668,18 @@ public function set_up_meta_fields() {
 				},
 			]
 		);
+		register_post_meta(
+			'course',
+			'_self_enrollment_not_allowed',
```

Comment:
We should be prefixing our meta keys as per https://senseip2.wordpress.com/programming-standards-best-practices/naming-conventions/#meta-keys.

Also, we need to delete this meta when we uninstall the plugin.

Reply:
> do you think it makes sense to also apply this standard to sensei post types?

I feel like we discussed this before but decided against it. Unfortunately I can't recall why. 😐

Reply:
No problem! Done in https://github.com/Automattic/sensei/pull/7231/commits/92d2086f91c92c65fb1db20eecfd2a47314f0508